src/DevTools/renderRelayTree.tsx

```diff
@@ -1,8 +1,18 @@
 import { LoadingClassName } from "Artsy/Relay/renderWithLoadProgress"
+import "DevTools/renderUntil"
+import { mount, RenderUntilCallback } from "enzyme"
 import React from "react"
 import { Variables } from "relay-runtime"
 import { MockRelayRenderer, MockRelayRendererProps } from "./MockRelayRenderer"
-import { renderUntil, RenderUntilCallback } from "./renderUntil"
+
+/**
+ * A {@link ReactWrapper.prototype.renderUntil} callback implementation that
+ * passes when no more loading indicators exist in the tree. Use this when you
+ * need to use `renderUntil` directly, such as after making updates to a Relay
+ * tree.
+ */
+export const RelayFinishedLoading: RenderUntilCallback = tree =>
+  !tree.find(`.${LoadingClassName}`).length
```

Comment:
@damassi So use this built-in callback to wait after triggering a refetch/pagination. I’d also appreciate it if you could add a code example once you’ve used it in a real test.

Reply:
Will do